packages/next/src/server/next-server.ts

```diff
@@ -711,6 +711,7 @@ export default class NextNodeServer extends BaseServer {
       require('next/dist/compiled/http-proxy') as typeof import('next/dist/compiled/http-proxy')
     const proxy = new HttpProxy({
       target,
+      followRedirects: true,
```

Comment:
Changing the default here could be considered a breaking change so I'm not sure it's ideal also users can now control proxying fully in middleware by returning a `fetch` in middleware so I'm not sure an experimental config here is needed either.